### DISCUSSION TITLE
Is there a way to use shares and recovery in web3auth plug and play?

### DISCUSSION THREAD

**Discussion by 0xall** [Web3Auth/Web3Auth#386] [Help]
Hi, I'm try to integrate web3auth in my app. :) I have some questions about integrating it.

1) I wanna use three shares. The first one is stored in my app, the second one is backup code that can be derived by user's security answers, and the last one is social login.

I found a way to implement it by using web3auth self host, but is there a way to use such that 3 shares in web3auth plug and play?

2) I want to know how to recover without social login in web3auth plug and play.

**Comment by yashovardhan** [comment]
Hey @0xall 

The splitting of 3 shares is happening on web3auth plug and play as well behind the scenes once you create a backup share (MFA). As long as you don't create a backup share, this off chain multi sig approach (ie. splitting of shares) is not enabled by default for the user. This is done to make sure a new user doesn't get lost and lose their account in the first go.
You can make it mandatory for them as well by changing the [multi factor authentication settings](https://web3auth.io/docs/sdk/web/openlogin#multi-factor-authentication).

Further, as i understand your question, it seems you want to remove social login feature of web3auth plug and play and only use the other 2 shares to login your user within the dApp. This is currently not supported in web3auth plug and play. You need to implement self host to enable this flow.
